package.json

```
@@ -127,4 +127,4 @@
     "not op_mini all"
   ],
   "snyk": true
-}
+}
```

```suggestion
}
```
Is this intentional?

---

nope this is introduced after running the `make version` command ( this change does not even show up in the local changes on vscode) - I can revert this if this seems weird